elibrary-micro/deployment.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.1'
+services:
+  app:
+    container_name: app-springboot-postgresql
+    image: app-springboot-postgresql
+    build: ./
+    ports:
+      - "8081:8080"
+    depends_on:
+      - dbpostgresql
+  dbpostgresql:
+    image: postgres
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=jfs-operations
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jfs-operations-app
+spec:
+  selector:
+    matchLabels:
+      app: jfs-operations-app
+  replicas: 5
+  template:
+    metadata:
+      labels:
+        app: jfs-operations-app
```

Comment:
Please change `app` to `elibrary-micro`.

Reply:
done